server/athenian/api/openapi/openapi.yaml

```diff
@@ -7320,3 +7377,242 @@ components:
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+        granularities:
+          items:
+            $ref: '#/components/schemas/Granularity'
+          type: array
+        quantiles:
+          $ref: '#/components/schemas/Quantiles'
+        account:
+          description: Session account ID.
+          type: integer
+        split_by_check_runs:
+          description: Calculate metrics separately for each number of check runs in suite.
+          type: boolean
+      required:
+      - account
+      - date_from
+      - date_to
+      - for
+      - granularities
+      - metrics
+      type: object
+    ForSetCodeChecks:
+      additionalProperties: false
+      description: Filter for `/metrics/code_checks`.
+      example:
+        repositories:
+        - github.com/athenianco/athenian-webapp
+        - github.com/athenianco/athenian-api
+        commit_authors:
+        - github.com/vmarkovtsev
+        - github.com/dpordomingo
+        jira:
+          labels_include:
+          - bug
+      properties:
+        repositories:
+          $ref: '#/components/schemas/RepositorySet'
+        repogroups:
+          $ref: '#/components/schemas/IndexedGroups'
+        commit_authors:
+          description: Check runs must be triggered by commits authored by these people.
+          items:
+            $ref: '#/components/schemas/DeveloperID'
+          type: array
+        commit_author_groups:
+          description: |-
+            Check runs must be triggered by commits authored by these people.
+            We aggregate by each group so that you can request metrics of several teams at once.
+            We treat `commit_authors` as another group, if specified.
+          items:
+            items:
+              $ref: '#/components/schemas/DeveloperID'
+            type: array
+          type: array
+        jira:
+          allOf:
+          - description: Analyze only those check runs that belong to PRs mapped to JIRA issues
+              that satisfy this filter.
+          - $ref: '#/components/schemas/JIRAFilter'
+      required:
+      - repositories
+      type: object
+    CodeCheckMetricID:
+      description: |-
+        * `chk-suites-count` - number of executed check suites.
+        * `chk-successful-suites-count` - number of successfully executed check suites.
+        * `chk-failed-suites-count` - number of failed check suites.
+        * `chk-cancelled-suites-count` - number of cancelled check suites.
+        * `chk-success-ratio` - number of successful check suites divided by `chk-suites-count`.
+        * `chk-suite-time` - average check suite execution time.
+        * `chk-suites-per-pr` - average number of times check suites executed in a pull request.
+        * `chk-suite-time-per-pr` - average check suite execution time by pull request. The difference with `chk-suite-time` is that we don't consider checks triggered by direct pushes, e.g. by merge commits.
```

Comment:
The main difference to me with `chk-suite-time` is more that `chk-suite-time-pr` can be relatively high is `chk-suite-per-pr` is high